v2/anomalydetection/frontend-setup.mdx

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_position: 2
+title: Frontend Setup
+hide_title: true
+---
+
+# Setting up the Frontend
+
+Setting up the frontend is relatively simple. You need to generate a request ID for each authentication event attempt. 
+This request ID is used as a parameter for the anomaly detection for detecting anomalies such as bot detection, impossible travel, and device information.
+
+Using it on other flows will not make a real difference, as, by their nature, other flows prevent this kind of anomalies.
+
+For detecting brute force attacks, password breaches, etc, you can skip these steps as the backend setup will handle this.
```

Comment:
It's quite confusing based on this statement. Should i do this step, or not? We need a clear indication in this page that this step is only required for Bot detection and impossible travel detection. Nothing else, right?

Reply:
done.